src/main/java/com/deep/sample/model/Problem.java

```diff
@@ -1,32 +1,60 @@
 package com.deep.sample.model;
 
-import com.fasterxml.jackson.databind.PropertyNamingStrategy;
-import com.fasterxml.jackson.databind.annotation.JsonNaming;
 
 public class Problem {
     private String title;
     private String descriptionValue;
 
+    /**
+     * it is constructor here.
+     */
+
```

Comment:
don't have additional space between method and javadoc

Reply:
completed